src/main/kotlin/org/rust/ide/inspections/lints/RsUnusedImportInspection.kt

```diff
@@ -89,6 +94,10 @@ class RsUnusedImportInspection : RsLintInspection() {
         )
     }
 
+    override fun createOptionsPanel(): JComponent = MultipleCheckboxOptionsPanel(this).apply {
+        addCheckbox("Ignore doctests", "ignoreDoctest")
```

Comment:
This name seems slightly confusing to me. Maybe something more along the line of `Disable inside doctests` or `Ignore unused imports in doctests`?